tests/matchers_unittest.py

```diff
@@ -133,6 +133,14 @@ def testAnyInstanceOf(self):
         self.assertEqual(testslide.matchers.AnyInstanceOf(str), "durrdurr")
         self.assertNotEqual(testslide.matchers.AnyInstanceOf(str), 7)
 
+    def testTriggeringAnyInstanceOfException(self):
+        c = sample_module.SomeClass()
+        with self.assertRaises(ValueError):
+            self.mock_callable(c, "method").for_call(
+                testslide.matchers.AnyInstanceOf(2)
+            ).to_return_value("mocked_response")
+            c.method()
```

Comment:
`testslide.matchers.AnyInstanceOf(2)` would raise the exception so invoking `c.method()` is not needed as well as creating the `c` instance is not neede.

Yu might want to have this test more inline with the other tests

```suggestion
        self.assertNotEqual(testslide.matchers.AnyInstanceOf(str), 7)
        with self.assertRaises(ValueError):
            testslide.matchers.AnyInstanceOf(2)
```